targets/csharp/UnittestRunner/UnittestRunner.csproj

```diff
@@ -42,7 +42,7 @@
     <DebugType>pdbonly</DebugType>
     <Optimize>true</Optimize>
     <OutputPath>bin\Release\</OutputPath>
-    <DefineConstants>TRACE;NETFX_CORE;SIMPLE_JSON_TYPEINFOENABLE_PLAYFABADMIN_API;ENABLE_PLAYFABSERVER_API;ENABLE_PLAYFABMATCHMAKER_API;ENABLE_PLAYFABENTITY_API</DefineConstants>
```

Comment:
These look like Unity pre-compiler defines. Are these csharp tests expected to run in that environment?